components/brave_shields/browser/ad_block_base_service.cc

```diff
@@ -33,70 +33,66 @@ using namespace net::registry_controlled_domains;  // NOLINT
 
 namespace {
 
-FilterOption ResourceTypeToFilterOption(content::ResourceType resource_type) {
-  FilterOption filter_option = FONoFilterOption;
+std::string ResourceTypeToString(content::ResourceType resource_type) {
+  std::string filter_option = "";
   switch (resource_type) {
     // top level page
     case content::ResourceType::kMainFrame:
-      filter_option = FODocument;
+      filter_option = "main_frame";
       break;
     // frame or iframe
     case content::ResourceType::kSubFrame:
-      filter_option = FOSubdocument;
+      filter_option = "sub_frame";
       break;
     // a CSS stylesheet
     case content::ResourceType::kStylesheet:
-      filter_option = FOStylesheet;
+      filter_option = "stylesheet";
       break;
     // an external script
     case content::ResourceType::kScript:
-      filter_option = FOScript;
+      filter_option = "script";
       break;
     // an image (jpg/gif/png/etc)
     case content::ResourceType::kFavicon:
     case content::ResourceType::kImage:
-      filter_option = FOImage;
+      filter_option = "image";
       break;
     // a font
     case content::ResourceType::kFontResource:
-      filter_option = FOFont;
+      filter_option = "font";
       break;
     // an "other" subresource.
     case content::ResourceType::kSubResource:
-      filter_option = FOOther;
+      filter_option = "other";
       break;
     // an object (or embed) tag for a plugin.
     case content::ResourceType::kObject:
-      filter_option = FOObject;
+      filter_option = "object";
       break;
     // a media resource.
     case content::ResourceType::kMedia:
-      filter_option = FOMedia;
+      filter_option = "media";
       break;
     // a XMLHttpRequest
     case content::ResourceType::kXhr:
-      filter_option = FOXmlHttpRequest;
+      filter_option = "xhr";
       break;
     // a ping request for <a ping>/sendBeacon.
     case content::ResourceType::kPing:
-      filter_option = FOPing;
+      filter_option = "ping";
       break;
-    // the main resource of a dedicated worker.
+    // the main resource of a dedicated
     case content::ResourceType::kWorker:
     // the main resource of a shared worker.
     case content::ResourceType::kSharedWorker:
     // an explicitly requested prefetch
     case content::ResourceType::kPrefetch:
     // the main resource of a service worker.
     case content::ResourceType::kServiceWorker:
-    // a report of Content Security Policy violations.
+    // a report of Content Security Policy
```

Comment:
This comment is missing the end too? Or maybe it's intentional, I can't tell. :-)